specs/conductor.md

```diff
@@ -41,11 +41,46 @@ The responsibilities of each module are as follows:
 
 ### Executor
  - Runs an event loop that handles receiving `ExecutorCommand`s from both the Driver and Reader
+ - Queues the incoming blocks to handle potential ordering issues and the basic fork choice rules 
+   different commit settings 
  - Filters out transactions by their rollup namespace and sends them to the rollup for execution (`soft` commits)
  - Catalogs and matches the block hashes received from the gossip network and the DA by rollup namespace to send `firm` commits to the rollup
  - Blocks are sent to the execution layer using [Astria’s GRPC Execution client interface](https://buf.build/astria/astria/docs/main:astria.execution.v1)
      - Rollups utilizing the Conductor must implement this interface
 
+#### The Execution Queue
+The Queue within the Executor is responsible for verifying the ordering of
+blocks received from the Sequencer. The basic flow for the blocks through the
+Executor and into and out of the Queue, is as follows:
+**From the Sequencer:**
```

Comment:
nit: if should be on new line, need two line breaks

Reply:
I removed the bold heading and reworded things a little and added the new lines.